### PR TITLE
Updated bidder

### PR DESCRIPTION
**Pull request by devmusings**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Other - Using a new endpoint
## Description of change

Using a new endpoint for the bidder and related code refactoring.
- contact email of the adapter’s maintainer: manohar@admedia.com
- [x] official adapter submission

### CODE REVIEW

File: src/adapters/admedia.js [hunk 1]
```diff
@@ -11,9 +11,29 @@ var CONSTANTS = require('../constants.json');
...
+      var request_obj = {};
+      request_obj.aid = aid;
+      request_obj.siteDomain = window.location.host;
+      request_obj.sitePage = window.location.href;
+      request_obj.siteRef = document.referrer;
+      request_obj.topUrl = utils.getTopWindowUrl();
+
+      var endpoint = ext_url+utils.parseQueryStringParameters(request_obj);
+
+      adloader.loadScript(endpoint, function(){ _processBids(params); }, true);
```
Comment (mkendall07) [inline src/adapters/admedia.js]: is this request meant to be cached? if not remove the 3rd param on `loadScript`

Comment (devmusings) [inline src/adapters/admedia.js]: Removed the parameter for caching.

**Comment by matthewlane** [comment]
Can you provide unit tests for these changes? Please see http://prebid.org/dev-docs/testing-prebid.html

**Comment by protonate** [comment]
Checking on status of this PR, any progress with unit tests?

**Comment by mkendall07** [comment]
Just checking back here @devmusings

**Pull request closed without merging**